cities/bremerhaven.json

```diff
@@ -3,9 +3,9 @@
     "features" : [
         {
             "properties" : {
-                "opening_hours" : "We 07:00-13:00; Sa 07:00-13:00",
                 "title" : "Wochenmarkt Geestemünde",
-                "location" : "Konrad-Adenauer-Platz"
+                "location" : "Konrad-Adenauer-Platz",
+                "opening_hours" : "We 07:00-13:00; Sa 07:00-13:00"
             },
```

Comment:
@jbein Can you please explain what is wrong about the order of properties?

Reply:
There is nothing wrong about the order. I just wanted to put "opening_hours" to the end of the list.

Reply:
Sorry. I thought I had answered already.

---

I took your commit message literally. I have to thoughts about this.

1) We currently don't have any sorting order defined or enforced. If you see a benefit I would like to ask you to describe it in an issue. I would love to see something that we can automatically apply to all city files.

2) Regarding this pull request: I would not mix changes to other cities in this pull request. It is in generell easier to review and discuss separate topics in separate pull request. This is just meant as a broad hint - since your changes are easy to review. No offense please.

Reply:
To resolve this I removed the sorting commit from this branch.